Task_1/utils/inspect_pickled_model.py

```diff
@@ -0,0 +1,22 @@
+import pickle
+
+# Path to the pickle file
+pickle_file_path = '/home/locolinux2/.local/workspace/checkpoint/experiment_109/best_model.pkl'
```

Comment:
This path (and anything similar) is going to be problematic. I will suggest using something that defined based on [`pathlib.Path.cwd()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.cwd) (or something similar) to ensure non-breakage.